Core/Object Arts/Dolphin/Installation Manager/DolphinProfessional.cls

```diff
@@ -78,6 +78,10 @@ contents
 		add: #('Core\Object Arts\Dolphin\MVP\Presenters\Prompters\Dolphin In-place Text Editor.pax' #encrypted #imageBased);
 		add: #('Core\Object Arts\Dolphin\IDE\Professional\Dolphin Refactoring Browser.pax' #encrypted #imageBased).
 
+	"Dolphin native Refactoring Browser integration"
```

Comment:
As stated in my overall comment, I do not think this should be loaded by default, nor stored under IDE/Professional